package.json

```diff
@@ -18,6 +18,7 @@
     "babel-runtime": "6.26.0",
     "css-loader": "0.28.7",
     "html-webpack-plugin": "2.29.0",
+    "obj-str": "^1.0.0",
```

Comment:
I accidentally deleted it in the last commit when trying to match CSSBlocks website dependencies